Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,58 @@ click: function () { closeClicked.raiseEvent(this); }');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('allowfullscreen', true);
+        infoElement.appendChild(frame);
 
         var viewModel = new InfoBoxViewModel();
-        this._viewModel = viewModel;
+        knockout.applyBindings(viewModel, infoElement);
+
+        //We inject default css into the content iframe,
+        //end users can remove it or add their own via the exposed frame property.
+        var cssLink = document.createElement("link");
+        cssLink.href = buildModuleUrl('Widgets/InfoBox/InfoBoxDescription.css');
+        cssLink.rel = "stylesheet";
+        cssLink.type = "text/css";
 
-        knockout.applyBindings(this._viewModel, infoElement);
+        //div to use for description content.
+        var frameContent = document.createElement("div");
+        frameContent.className = 'cesium-infoBox-description';
+
+        this._container = container;
+        this._element = infoElement;
+        this._frame = frame;
+        this._viewModel = viewModel;
+        this._descriptionSubscription = undefined;
+
+        var that = this;
+
+        //We can't actually add anything into the frame until the load event is fired
+        frame.addEventListener('load', function() {
+            var frameDocument = frame.contentDocument;
+            frameDocument.head.appendChild(cssLink);
+            frameDocument.body.appendChild(frameContent);
+
+            //We manually subscribe to the description event rather than through a binding for two reasons.
+            //1. It's an easy way to ensure order of operation so that we can adjust the height.
+            //2. Knockout does not bind to elements inside of an iFrame, so we would have to apply a second binding
+            //   model anyway.
+            that._descriptionSubscription = subscribeAndEvaluate(viewModel, '_descriptionSanitizedHtml', function(value) {
+                // Set the frame to small height, force vertical scroll bar to appear, and text to wrap accordingly.
+                frame.style.height = '5px';
+                frameContent.innerHTML = value;
+                // Measure and set the new custom height, based on text wrapped above.
+                frame.style.height = frameContent.getBoundingClientRect().height + 'px';
+            });
+
+            //Once the frame is initialized, we can enable sandboxing.
+            frame.setAttribute('sandbox', 'allow-popups allow-forms'); // allow-same-origin allow-pointer-lock allow-scripts allow-top-navigation
```

Comment:
The problem now is that the HTML is evaluated before the sandbox attribute is added, which means it will run with full access.